### ISSUE TITLE
modify readme

### ISSUE THREAD

**Issue by av1shek** [av1shek/All-Coding-Platform-Solutions#3]
Create an attractive readme file describing that we are accepting pull requests for Hacktoberfest 2021. Also, describe some rules about how to contribute and file name format.
You can decide the rule, but it should be non-conflicting and easy to navigate.

**Comment by ritugupta02** [comment]
can u assign this issue to me?

**Comment by dollygupta07** [comment]
Hi,
Is there any other issue in this repo I can work on?

**Comment by av1shek** [comment]
> Hi, Is there any other issue in this repo I can work on?

@dollygupta07 You can solve issue #17.

**Comment by dollygupta07** [comment]
> > Hi, Is there any other issue in this repo I can work on?
> 
> @dollygupta07 You can solve issue #17. If you are interested you can ask to assign that issue in the comments there.

Yes, I'm interested. If this issue is still open, can you assign it to me?
